app/views/holds/_holds_not_ready_table.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with url: '/holds/multiple', method: :delete do %>
+<%= form_with url: '/batch_update', method: :delete, id: 'pending-holds' do %>
```

Comment:
changed the path/method name just to make it clearer in my head

Reply:
what do you say for sth like `/holds/batch_update`?

Reply:
yeah that's better, I'll change it

Reply:
gotta update the route too and stuff, more to it